src/clr-angular/utils/drag-and-drop/if-dragged.spec.ts

```diff
@@ -74,7 +82,14 @@ export default function(): void {
     });
 }
 
-@Component({template: `<div clrDraggable #draggable><div *clrIfDragged>Test</div></div>`})
-class IfDraggedTest {
-    @ViewChild("draggable") draggableEl: ElementRef;
+@Component({providers: [ClrDragEventListener], template: `<div *clrIfDragged>Test</div>`})
+class IfDraggedTest<T> {
+    public draggableEl: Node;
+    constructor(private el: ElementRef, private dragEventListener: ClrDragEventListener<T>) {
+        this.draggableEl = el.nativeElement;
+        dragEventListener.attachDragListeners(this.draggableEl);
```

Comment:
You misunderstood me, it's not that I didn't want to use the `clrDraggable` specifically, it's that I didn't want to test stuff that you already test in another spec, using low-level mouse and touch events. With this latest version, if `ClrDragEventListener` has a bug, all the tests from this spec will fail too...

I realize `DragEventListener` doesn't offer a way to force a dragStart or dragEnd broadcast directly, so you can write a dummy mock that just exposes the two Subjects for instance. The whole point is for you to test that:
> When `ClrDragEventListener` emits `dragStarted`, the ghost instantiates the template.
not that:
> When user presses the left mouse button and starts moving the mouse, the ghost instantiates the template.

Reply:
Implemented the requested change. Please take a look again.